chapters/en/slides/module1/module1_02.Rmd

```diff
@@ -0,0 +1,280 @@
+such as the x-axis, y-axis, color, *etc*.
+The rest of the plot details are handled automatically.
+
+---
+
+## The Python plotting landscape
+
+<img src="/module1/py-plotting-landscape.png" alt="The Python plotting landscape" width="100%"></img>
+
+Notes:
+In this image you can see the most commonly used Python plotting packages.
+There are many more,
+but these are the ones you are the most likely to hear about,
+so it is good to know that they exist.
+As you can see there are several high and low level language,
+so how to we chose?
+In this course we have chosen to use Altair,
+because it is a powerful high level visualization tool 
+with a clear and consistent syntax
+that also allows us to add interactive components to our plots,
+such as tooltips and selections.
+
+---
+
+## A high level grammar of graphics helps us compose plots effectively
+
+- Simple grammatical components combine to create visualizations.
+- The Altair visualization grammar consist of three main components:
+    1. Create a chart.
+    2. Add a graphical mark.
+    3. Encode dataframe columns as visual channels.
+- In code, it looks like this: `Chart(data).mark().encode(x, y, ...)`.
+
+Notes:
+The declarative plotting concept can be implemented in different ways.
+Here we will describe how a visualization grammar
+can be used similarly to how grammar in regular language
+is used to build complex linguistic constructs.
+A wide range of simple to sophisticated visualizations can be created using a concise grammar.
+Thanks to this functional way of interfacing with data,
+only minimal changes are required if the underlying data change or to change the type of plot.
+
+One of the most prominent declarative statistical visualization libraries is [Altair](https://altair-viz.github.io/).
+Altair offer a powerful and concise visualization grammar for quickly building a wide range of statistical graphics.
+In brief, you first create a chart,
+then you indicate which graphical marks should represent the data (points, lines, etc)
+and finally you encode your dataframe colums as different visual channels (x, y, color, etc).
+
+Enough talking,
+let's code!
+
+---
+
+## Sample data can be found in Altair's companion package vega_datasets
+
+```{python}
+from vega_datasets import data
+
+cars = data.cars()
+cars
+```
+
+Notes:
+When specifying data in Altair,
+it is important that dataframes are in the ["tidy"](http://vita.had.co.nz/papers/tidy-data.html) format.
+They should consist of a set of named data *columns* (also called *fields* or *variables*)
+with one feature each
+and rows with one *observation* each.
+In our Altair examples
+we will often use data from the [vega-datasets](https://github.com/vega/vega-datasets) package.
+Many of these datasets are directly available as Pandas dataframes,
+such as the `cars` dataset.
+
+---
+
+## Let's create our first plot
+
+```{python}
+import altair as alt
```

Comment:
Let's talk about this in the transcript. It might be a while since their programming in Python course, so a reminder what this is doing here, and that altair is the name of the package we are importing and alt is just the nickname/alias would probably be welcome by the learners.

Reply:
Done